src/dimension.c

```diff
@@ -51,7 +55,7 @@ ts_hyperspace_get_dimension_by_id(Hyperspace *hs, int32 id)
 	return bsearch(&dim, hs->dimensions, hs->num_dimensions, sizeof(Dimension), cmp_dimension_id);
 }
 
-Dimension *
+TSDLLEXPORT Dimension *
```

Comment:
Unnecessary change and it is good to revert:
```suggestion
Dimension *
```